types/src/toGamut.d.ts

```diff
@@ -14,3 +14,15 @@ declare function toGamut(
 ): PlainColorObject;
 
 export default toGamut;
+
+declare namespace toGamutCSS {
+	let returns: "color";
+}
```

Comment:
This isn't necessary for methods unless they're registered on Color objects (e.g. `myColor.toGamutCSS()`).